OpenMarket/OpenMarket/Model/Network/DataEncoder.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  DataEncoder.swift
+//  OpenMarket
+//
+//  Created by 우롱차 on 2022/06/08.
+//
+
+import UIKit
+
+struct DataEncoder {
+
+    private let imageEncoder: ImageEncoder
+    private let jsonEncoder: JSONEncoder
+    private var boundary: String
+    private var boundaryPrefix: String {
+        get {
+            return "\r\n--\(boundary)\r\n"
+        }
+    }
+    
+    init(boundary: String){
+        imageEncoder = ImageEncoder()
+        jsonEncoder = JSONEncoder()
+        self.boundary = boundary
+    }
+    
+    init(imageEncoder: ImageEncoder, jsonEncoder: JSONEncoder, boundary: String) {
+        self.imageEncoder = imageEncoder
+        self.jsonEncoder = jsonEncoder
+        self.boundary = boundary
+    }
```

Comment:
`boundary`만 설정하고 나머지는 기본값을 사용하기위해 생성자를 따로 구현하신 건가요? 🤔 
```suggestion
    init(imageEncoder: ImageEncoder = ImageEncoder(), jsonEncoder: JSONEncoder = JSONEncoder(), boundary: String) {
        self.imageEncoder = imageEncoder
        self.jsonEncoder = jsonEncoder
        self.boundary = boundary
    }
```
생성자를 두 개 만들기보단, 인자의 기본값을 활용하면 좀 더 간단하게 표현할 수 있을 것 같아요.

Reply:
수정하도록 하겠습니다! 그런데 이때 변수들의 위치는 상관 없을까요? default 값이 없는게 가장 앞에 오는게 좋을까요???

Reply:
위치는 어느 것이 앞에 오든 차이 없을 것 같아요.